cvat-ui/src/components/task-page/user-selector.tsx

```diff
@@ -50,7 +83,10 @@ export default function UserSelector(props: Props): JSX.Element {
     const autocompleteRef = useRef<RefSelectProps | null>(null);
 
     useEffect(() => {
-        core.users.get({ limit: 10, is_active: true }).then((result: User[]) => {
+        const state = getCVATStore().getState();
+        const userID = state.auth.user?.id;
+        const organizationSLUG = state.organizations.current?.slug;
+        initialUsersStorage.get(userID, organizationSLUG).then((result: User[]) => {
```

Comment:
IMHO using `organizationSlug` is better. Uppercase here breaks camel case style. Its ok for ID cuz its abbreviation.